frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -100,530 +121,536 @@ export function FeatureFlag(): JSX.Element {
     return (
         <div className="feature-flag">
             {featureFlag ? (
-                <Form
-                    layout="vertical"
-                    form={form}
-                    initialValues={{ name: featureFlag.name, key: featureFlag.key, active: featureFlag.active }}
-                    onValuesChange={(newValues) => {
-                        if (featureFlagId !== 'new' && newValues.key) {
-                            setHasKeyChanged(newValues.key !== featureFlag.key)
+                <>
+                    <FeatureFlagPageHeader activeTab={FeatureFlagTab.General} id={featureFlag.id} />
```

Comment:
Nit: we could save ~1000 lines of diff, if instead of `<>` you made the `<FeatureFlagPageHeader />` always visible. It might as well be --> seeing some UI could be better?